web/business-registry-dashboard/.env.sample

```diff
@@ -5,7 +5,7 @@ NUXT_REGISTRIES_SEARCH_API_KEY=""
 NUXT_X_API_KEY=""
 
 # vault API (auth-api)
-NUXT_AUTH_API_URL="https://auth-api-dev-142173140222.northamerica-northeast1.run.app"
+NUXT_AUTH_API_URL="https://test.api.connect.gov.bc.ca/auth-dev"
 NUXT_AUTH_API_GW_URL="https://test.api.connect.gov.bc.ca/auth-dev"
```

Comment:
I believe AUTH_API_URL is used by the core layer while AUTH_API_GW_URL is used by this app.